app/cypress/integration/state.spec.js

```diff
@@ -159,3 +159,67 @@ describe("State page Water Quality Overview sub tabs", () => {
       .should("exist");
   });
 });
+
+describe("State page Water Overview tab", () => {
+  beforeEach(() => {
+    cy.visit("/state/AK/water-quality-overview");
+  });
+
+  it(`Display "Drinking Water Information" when water sub-tab clicked on`, () => {
+    cy.findByText("Drinking Water").click();
+    cy.findAllByText(/Drinking Water Information for/)
+      .filter(":visible")
+      .should("exist");
+  });
+
+  it(`Clicking "Expand All/Collapse All" expands/collapses the state documents and state water stories menu`, () => {
+    const documentsText = "Documents Related to Integrated Report";
+    const waterText =
+      "Upgrading Boat Motors Reduces Hydrocarbon Pollution in Kenai River";
+
+    cy.findAllByText("Expand All").filter(":visible");
+    cy.findByText(documentsText).should("not.exist");
+    cy.findByText(waterText).should("not.exist");
+
+    cy.findByText("Expand All").click();
+    cy.findAllByText(documentsText).should("be.visible");
+    cy.findAllByText(waterText).should("be.visible");
+
+    cy.findByText("Collapse All").click();
+    cy.findByText(documentsText).should("not.exist");
+    cy.findByText(waterText).should("not.exist");
+  });
+
+  it(`Clicking "<state name> Documents" opens the documents content`, () => {
+    const title = "Alaska Documents";
+    const text = "Documents Related to Integrated Report";
+
+    // verify text is not visible
+    cy.findByText(text).should("not.be.visible");
+
+    // open accordion and check text is visible
+    cy.get(".hmw-accordion").contains(title).click();
```

Comment:
Normally I'd recommend `cypress-testing-library` selectors over `cy.get()` (with the exception of re-selecting Cypress aliases), but I see the issue w/ `findByText()` is it won't find the matched text because "Alaska" is in a `<strong>` tag, and findByText just searches a node's `textContent`... (the error message is helpful: "This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.")

`*-testing-library`'s handling of this situation is pretty frustrating, as it seems to go against the spirit of their recommendation to write tests that replicate how a user uses the app. The user doesn't care if some text is in another DOM element, as long as they can read it...

In any case, I think I'm fine your use of `cy.get()` here, as it gets around this issue, and we're not likely to remove the `hmw-accordion` class name. The alternative would be to write a helper function that looks inside an element's children node's textContent for a match, but I'm not sure that's worth it. I'll ping @oftheheadland and @cschwinderg here to get their take as well. I'm good with whatever the team collectively decides here – it may be practicality wins, as this does correctly test the test case.

Reply:
I'm good with the way it is now.

Reply:
I'm okay with it too 👍

Reply:
Thanks for weighing in @cschwinderg and @oftheheadland.